index.html

```diff
@@ -123,13 +119,11 @@ <h2>Photo Gallery</h2>
                         </span>
                     </blockquote>
                 <p id="page-copyright">&copy; 2019 <a href="mailto:Brett.Marshall@student.saultcollege.ca?subject=Messsage%20from%20CSD120%20Website">Brett Marshall</a></p>
-                <picture>
-                <source media="(min-width:1200px)" srcset="images/sc-logo@4x.png">
-                <source media="(min-width:1000px)" srcset="images/sc-logo@3x.png">
-                <source media="(min-width:800px)" srcset="images/sc-logo@2x.png">
-                <source media="(min-width:600px)" srcset="images/sc-logo@1.5x.png">
-                <img src="images/sc-logo@.png" alt="Sault College Logo">
-                </picture>
+                <img class="sc-logo" 
+                    src="images/sc-logo.png" 
+                    sizes="70px" 
+                    srcset="images/sc-logo.png 70w, images/sc-logo@1.5x.png 105w, images/sc-logo@2x.png 140w, images/sc-logo@3x.png 210w, images/sc-logo@4x.png 280w" 
+                    alt="Sault College Logo">
```

Comment:
The `<picture>` element isn't what you want here.  In order to have the browser pick the most appropriate image based on the size of available images, the screen size & density, and the desired image size, you need to use the sizes/srcset technique as shown here